en/docs/learn/configuring-just-in-time-provisioning-for-an-identity-provider.md

```diff
@@ -100,7 +100,23 @@ below:
             store
             configuration](../../setup/configuring-user-stores)
             .
-        
+
+        !!! note
```

Comment:
Shall we make this a bullet point instead of a note. It can go like this:

```
- Associate JIT provisioned user with local user
    
    !!! note
         This configuration is available as an update in WSO2 IS 5.11.0 from update level 69 onwards 
            (Updates 2.0 model). If you don't already have this update, see the instructions on 
            [updating your product](https://updates.docs.wso2.com/en/latest/updates/overview/).
         
   To associate the JIT provisioned user...........
```

wdyt?

Reply:
Addressed with commit 3ab0f98903f9420d8a3dac66a629df96149596b8